src/routes.js

```diff
@@ -24,6 +24,7 @@ import CollaboratorsCreate from './routes/Collaborators/Create';
 import CollaboratorView from './routes/Collaborators/View';
 import CollaboratorEdit from './routes/Collaborators/Edit';
 import Gigs from './routes/Gigs';
+import GigsView from '../src/routes/Gigs/View/GigsView';
```

Comment:
can be simplified, because there is index file in Gigs/View directory (`src/routes/Gigs/View/index.js`), this import can be

`import GigsView from '../src/routes/Gigs/View';`

Reply:
amended.